src/core/render/utils.js

```diff
@@ -23,16 +23,18 @@ export function getAndRemoveConfig(str = '') {
 
   if (str) {
     str = str
-      .replace(/^('|")/, '')
-      .replace(/('|")$/, '')
-      .replace(/(?:^|\s):([\w-]+:?)=?([\w-%]+)?/g, (m, key, value) => {
-        if (key.indexOf(':') === -1) {
-          config[key] = (value && value.replace(/&quot;/g, '')) || true;
-          return '';
-        }
+      .replace(
+        /(?:^|\s):([\w-]+:?)=?([\w-%]+|&quot;((?!&quot;).)*&quot;|[“”][^“”]*[“”])?/g, // Note: because the provided `str` argument has been html-escaped, with backslashes stripped, we cannot support escaped characters in quoted strings :-(
+        function (m, key, value) {
```

Comment:
FYI, there is a bit complicated regex including `negative lookahead`, we may need more tests here.

Reply:
Yes, the regex became more complicated because the incomming `str` argument has been transformed from markdown to html. This is also the reason we cannot support escaped characters in the quoted-string config-values. It would have been a lot cleaner if the markdown to html transformation would have been applied after the config strings would have been read.